### DISCUSSION TITLE
A question about certificates when using the maven dependency

### DISCUSSION THREAD

**Discussion by alfchristensen** [OxalisCommunity/oxalis#630] [Q&A]
Hi!

A 3rd party certificate is mandatory when setting up an Access Point.

When using the maven dependency of Oxalis only for sending documents, is it still necessary to use a 3rd party certificate?

Best Regards
Alf Christensen

**Comment by dladlk** [comment]
Can you rephrase your question - it is not clear what do you ask about. E.g. put a link to a documentation where you got the first statement.

**Reply by alfchristensen** [reply]
Sorry, the 3rd party certificate is a requirement from the testbed, not Oxalis. You can delete this discussion if you want, as it will probably only confuse others.
